mathlibtools/leanproject.py

```diff
@@ -3,7 +3,6 @@
 from datetime import datetime
 from typing import Tuple, Optional
 from getpass import getpass
-
```

Comment:
This blank line is intentional, it separates the stdlib imports from third-part ones